src/api/posts/test/feeds.test.js

```diff
@@ -37,3 +37,27 @@ describe('Test GET /feeds/invalid endpoint', () => {
     expect(res.body[0].id).toBe(feeds[0].id);
   });
 });
+
+describe('GET /feeds/info', () => {
+  test('Should return 200 and valid response object', async () => {
+    function checkKeys(resBody) {
+      let bool = true;
+      const allKeys = ['waiting', 'active', 'completed', 'failed', 'delayed', 'paused', 'jobCnt'];
+      const resKeys = Object.keys(resBody.queueInfo);
+      for (let i = 0; i < resKeys.length; i += 1) {
+        if (allKeys.indexOf(resKeys[i]) < 0 || typeof resBody.queueInfo[resKeys[i]] !== 'number') {
+          bool = false;
+          break;
+        }
+      }
```

Comment:
```suggestion
      Object.keys(resBody.queueInfo).forEach((key) => ;
        if (!allKeys.includes(key) || typeof key !== 'number') 
          return false;
```
(didn't test it, but it's probably something very close to this)

Reply:
I used `every` instead of `forEach` because `forEach` doesn't support break or return